subworkflows/local/allele_specific_calling.nf

```diff
@@ -43,7 +44,14 @@ workflow ALLELE_SPECIFIC_CALLING {
         )
         ch_versions = ch_versions.mix(GATK4_ASEREADCOUNTER.out.versions.first())
 
+        BOOTSTRAPANN(
+            BCFTOOLS_VIEW.out.vcf.join(BCFTOOLS_INDEX.out.tbi),
```

Comment:
```suggestion
            vcf_tbi,
```

Reply:
I think you want to annotate the original vcf. Non-heterozygote variants in the original vcf will simply not get a annotation.

Reply:
You are right, I was still thinking about the asereadcounter